app/css/custom.scss

```diff
@@ -233,14 +283,43 @@ code {
     flex: 1 0 auto;
   }
   header {
-    z-index: 1;
+    z-index: 2;
     top: 0px;
     left: 0px;
     right: 0px;
+    nav {
+      .item.logo-wrapper {
+        padding: 0;
+      }
+      .ui.menu {
+        background-color: $primary-color;
+        border-radius: 0;
+        a, i {
+          color: #fff;
+        }
+        .right.menu {
+          .item {
```

Comment:
This is so true. I just got back from a research on the subject matter and they say a maximum of 3-levels. ( https://css-tricks.com/sass-style-guide/ ).

Thanks for this tip. We'd refractor.